src/gatsby-node.js

```diff
@@ -47,25 +47,37 @@ exports.sourceNodes = (
     return nodeData
   }
 
-  const { groupUrlName, ...apiOptions } = configOptions
+  const { groupUrlName, eventsOptions: paramEventsOptions, ...apiOptions } = configOptions
+  const eventsOptions = paramEventsOptions?paramEventsOptions:[apiOptions];
+
   // Convert the options object into a query string
   const queryStringOptions = queryString.stringify(apiOptions)
 
-  const apiGroupUrl = `https://api.meetup.com/${groupUrlName}?${queryStringOptions}`
-  const apiEventsUrl = `https://api.meetup.com/${groupUrlName}/events?${queryStringOptions}`
+  const apiGroupUrl = `https://api.meetup.com/${groupUrlName}?${queryStringOptions}`; // Convert the options object into a query string
+
+  const allApiEventsUrl = eventsOptions.map(eventApiOptions => {
```

Comment:
if the `eventsOptions` parameter is null, this looks like it will raise an exception. Could you please make sure that `eventsOptions` parameter is fully optional, so that we don’t break backward compatibility?

Reply:
I think I've already made sure that eventsOptions parameters is fully optional, the current line after get groupUrlName:
`const eventsOptions = paramEventsOptions?paramEventsOptions:[apiOptions];`
But I will test it again with more tests to verify it.

Reply:
@phacks , I added tests to verify that eventsOptions is optional.